### PR TITLE
feat : Added pages to login and signup

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Login page for registered user and signup page for new users.

### CODE REVIEW

File: src/Components/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    <input type="checkbox" /> Remember me
+                   </label>
+                <p className="forgot-pass">
+                    <a href="/" className="forgot-pass-link" >
+                        Forgot your Password?
+                    </a>
+                </p>               
+               </div>
+                <button className="submit-form-btn primary-btn btn btn-login"> Log In </button>
+            </form>
```
Comment (Jia-A) [inline src/Components/Login/Login.jsx:30]: Good use of semantic HTML

File: src/Components/Login/login.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+}
+.new-user{
+    font-size: var(--text-size-m);
+    text-align: center;
+    padding:1rem;
+}
+
+.forgot-pass-link{
+    color: var(--primary-color);
```
Comment (Jia-A) [inline src/Components/Login/login.css:63]: Good use of CSS variables

**Review by Jia-A — APPROVED** [review]
Everything looks good, you can merge it.

**Review by stutisk — APPROVED** [review]
extra spaces can be removed
looks good to merge

**Pull request merged**